web/src/components/DLP/OverviewTab.vue

```diff
@@ -290,19 +316,33 @@ export default defineComponent({
         : Math.min(Object.keys(assetSummary.value).length, 3)),
     );
 
+    const assetSummaryBeingComputed = computed<boolean>(() => currentDandiset.value?.status === 'Pending');
+
     const contactPeople = computed(
       () => new Set(contributors.value
         .filter((contributor) => contributor.roleName?.includes('dcite:ContactPerson'))
         .map((contributor) => contributor.name)),
     );
 
+    onMounted(() => {
+      setInterval(async () => {
+        if (!currentDandiset.value || !assetSummaryBeingComputed.value) {
+          return;
+        }
+        const { identifier } = currentDandiset.value.dandiset;
+        const { version } = currentDandiset.value;
+        await store.fetchDandiset({ identifier, version });
+      }, 5000);
+    });
```

Comment:
We should probably add a stop condition where we call `clearInterval`, so we're not needlessly making the same request every 5 seconds after the fact.

Reply:
Once the assets summary is computed, `assetSummaryBeingComputed` will be `true`, so the request won't happen due to the if guard.

It's also possible that `assetSummaryBeingComputed` is initially `true` on page load, but then gets set to `false` due to something the user does on the DLP (for example, editing metadata in the meditor), so I don't think we'd want to do a `clearInterval` at any point here.

Reply:
Just so I understand: the request wouldn't fire, but the interval callback would continue to run every 5 seconds, right? Would we still not want to call `clearInterval()` just before the `return` statement on line 330?

Reply:
> Just so I understand: the request wouldn't fire, but the interval callback would continue to run every 5 seconds, right? 

Correct.

> Would we still not want to call clearInterval() just before the return statement on line 330?

No, because it's possible that the user will make some change on the DLP that will cause the assets summary to become outdated. In that case, if we had called clearInterval prior, we would need to call `setInterval` again. We _could_ do that, but that would require figuring out all the ways the user can retrigger assets summary calculation from the DLP, and calling setInterval from each of those places. Then that introduces more issues, like we'd need some way to tell if the setInterval call is already running to avoid duplicating it, etc. So I think just leaving it as-is is the cleanest way to do this.